server.js

```diff
@@ -23,3 +23,38 @@ app.get('/api/v1/all-products', async (request, response) => {
   }
 });
 
+
+// app.post('/api/v1/favorite-product', async (request, response) => {
+//   const product = request.body;
+
+//   if(!product.id) {
+//     response
+//       .status(422)
+//       .send({ error: 'Id is required.'});
+//   }
+
+//   try {
+//     const id = await database('product').insert(product, 'id');
+//     response.status(201).json({ id })
+//   } catch (error) {
+//     response.status(500).json({ error });
+//   }
+// });
+
+app.put('/api/v1/favorite/:id', async (request, response) => {
```

Comment:
What do you think about refactoring the url to `favorites` since we're assuming that they'll be more than 1?

Reply:
I went for the name favorite due to this end point only manipulating one item when it's hit. I felt that favorite might be more descriptive than favorites due to it only handling one thing. What are your thoughts?

Reply:
That makes sense, thank you for the clarification.